1.8/overview/concepts.md

```diff
@@ -1,121 +1,282 @@
 ---
-post_title: Concepts
+post_title: DC/OS Architecture Concepts
 nav_title: Concepts
 menu_order: 5
 ---
-This page contains terms and definitions for DC/OS.
+## <a name="dcos-concepts"></a>DC/OS Concepts
 
-# <a name="adminrouter"></a>Admin Router
+DC/OS is an abstraction around pre-existing components. These terms are sometimes similar to component terms and sometimes they’re new terms meant to abstract organically grown component terms.
```

Comment:
will do. Tho i might tweak that last bit. "used in a different way" isn't quite what i meant. some of the terms are new and hide component terms with a new abstraction layer.